apiary.apib

```diff
@@ -74,31 +74,37 @@ Any response that does not provide a ```200 OK``` response code should be treate
 # Group Master Data
 The master data collection allows for integration between the Dude Platform and Dude Solutions' other system of records.
 
-## Get User [/masterdata/users/{userName}]
+## Get User [/masterdata/users/{userName}{?sourceAbbreviation}]
 The get user is exposed and provided to return a Dude Platform user from a system of record user name.
 Any response that does not provide a ```200 OK``` response should be treated as an error.
+
 + Parameters
-    + `userName` (string, required) - The name of the user from the system of record.
+    + userName (string, required) - The name of the user from the system of record.
+    + sourceAbbreviation (string, required) - The system of record source abbreviation (SD or FD).
```

Comment:
its a querystring but yes it is required.....these routes need changed but for now this is what we got